packages/storybook/stories/Examples/IncomingCallAlerts/components/IncomingCallAlerts.tsx

```diff
@@ -57,12 +57,12 @@ export const IncomingCallToast = (props: IncomingCallToastProps): JSX.Element =>
         <DefaultButton
           className={incomingCallRejectButtonStyle}
           onClick={() => onClickReject()}
-          onRenderIcon={() => <CallEnd20Filled />}
+          onRenderIcon={() => <CallEnd20Filled style={{ height: '1.25rem' }} />}
         />
         <DefaultButton
           className={incomingCallAcceptButtonStyle}
           onClick={() => onClickAccept()}
-          onRenderIcon={() => <Call20Filled />}
+          onRenderIcon={() => <Call20Filled style={{ height: '1.25rem' }} />}
```

Comment:
Looks good to me. Just wondering if this change in icon was all it took to center it?

Reply:
That's right, 
I actually changed it to use "display: 'flex'" as it is less obtrusive if Contoso ever decided to change height dimensions.